### ISSUE TITLE
Enhance documentation for sharing CephFS volumes across namespaces

### ISSUE THREAD

**Issue by guimou** [rook/rook#10370]
An interesting use case for CephFS volumes is to store shared elements: libraries of applications, documentation, project templates, shared data... Those volumes can be mounted as ReadOnlyMany (ROX) in multiple pods, or even ReadWriteMany (RWX) to create a shared workspace.

However, as PVCs are scoped to a namespace, it's impossible to simply share a PVC-based library between different namespaces. If you have different applications in different namespaces that would make use of such a shared library, you are forced to duplicate the library/volume in each and every namespace, which kinds of defeats the purpose.

To solve this scoping issue, there is a way to create a PVC based on CephFS, then manually create PVs and PVCs in different namespaces that would connect in fact to the same CephFS volume, achieving the goal of a truly shared library at the cluster level.

This feature request is to enhance the documentation to clearly expose the steps on how to achieve this.

**Comment by travisn** [comment]
This is a frequent question for users, how to configure a shared filesystem PVC across namespaces. Since this can be accomplished with a static CephFS PV, we should document how this can be done. I'm thinking this can be a new section in the [filesystem-storage.md](https://rook.github.io/docs/rook/v1.9/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage/) topic. @Rakshith-R Could you pick this up? Thanks!

**Comment by Madhu-1** [comment]
* create a PVC
* get the PV for the PVC and extract the volumeHandle
* Get the copy of PV and change the name of the PV and change reclaimPolicy to Retain (add not about PV deletion here we need to keep the main PVC until all static PVC's are deleted or change reclaimPolicy of the main PVC to Retain also)
* create more PV with new names
* Create more PVC in different namespaces and point PV in the PVC.spec.VolumeName

The above steps should help for this use case.

**Comment by guimou** [comment]
I followed the procedure and it seems to work well, although there is a small problem when I want to use the shared PVC in readonly mode in a pod (it's a shared library, so I don't want people to mess with it).
Here is what I did: 
- I created a PVC in namespace 1, which created a PV (PV-base)
- I created another PV with another name (PV-base-copy) pointing to the same CephFS volume
- In namespace 2, I created a PVC using PV-base-copy (basically copied the first PVC content and changed the PV it pointed to). PVC got bound ok.
- When I launch a pod in namespace 2 using this PVC, the Attach.Volume command for the PVC works: `AttachVolume.Attach succeeded for volume "pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-opendatahub"`
- But then in the events I also get: `MountVolume.NodeExpandVolume failed for volume "pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-opendatahub" requested read-only file system`
The volume is indeed defined as ReadOnly in the pod as it's a share library, so I don't want people to mess with it:
```
volumes:
    - name: easybuild-data
      persistentVolumeClaim:
        claimName: easybuild-data
        readOnly: true
      mountPath: /opt/apps/easybuild
```
When I put `readOnly: false` everything works well.

Btw, I can totally help review or even create the doc for this as I have both hands in it anyway.
